src/commoncode/urn.py

```diff
@@ -25,6 +25,8 @@
 from __future__ import absolute_import
 from __future__ import print_function
 from __future__ import unicode_literals
+from future.standard_library import install_aliases
```

Comment:
What does this do?

Reply:
This module has been used to import urlib module in python 2/3. Refer to this documentation of future module.
Reference: https://python-future.org/compatible_idioms.html#urllib-module